src/npc/dialog.py

```diff
@@ -40,7 +40,7 @@ def __init__(self, character_name: str, text: str, font: pygame.Font):
         :param text: The dialogue the character is supposed to say.
         :param font: The font used to render this dialogue."""
         super().__init__()
-        self.z: int = TB_LAYER
+        self.z: int = LAYERS['text box']
```

Comment:
I put that value in a constant because it wasn’t needed elsewhere, which is why I will ask you to revert that change

Reply:
I think we just have different ideas of what clear code should be, but okay, I’ll revert